apps/composable/app.json

```diff
@@ -0,0 +1,10 @@
+{
+    "appCategory": "DeFi",
+    "appName": "Composable", 
+    "appSummary": "Composing DeFi for Mass Adoption - Any money, any chain, anywhere",
+    "appWebsiteUrl": "https://app.trustless.zone/",
```

Comment:
https://www.composable.finance/`
can we use this please thanks!

Reply:
@josietyleung hmm I'm debating between https://www.composable.finance/apps and the one in the script. fyi appWebsiteUrl is the link that lead to composable app. What do you think ?

Reply:
IMO, Composable is more like a tool or infrastructure and https://app.trustless.zone is a DeFi app made of using Composable. So, `appWebsiteUrl` should be https://www.composable.finance` and `appCategory` should be `Tool`. How about this?

Reply:
I agree @josietyleung wdyt ?

Reply:
yes sounds good thanks!